test/jdk/ProblemList-Virtual.txt

```diff
@@ -44,8 +44,6 @@ javax/management/remote/mandatory/loading/MissingClassTest.java 8145413 windows-
 
 javax/management/remote/mandatory/loading/RMIDownloadTest.java 8308366 windows-x64
 
-java/lang/ScopedValue/StressStackOverflow.java 8309646 generic-all
-
```

Comment:
That's the issue for the JTREG_TEST_THREAD_FACTORY=Virtual runs. I think we need to leave it excluded until that issue is diagnosed/fixed. 

However, in jdk/test/ProblemList.txt, it is excluded on linux-s390x as that platform doesn't seem to have, or use, VM continuations and so is using the fallback/alternative implementation. I think this entry can be removed now.

Reply:
I think it's fixed now.

Two problems:
1. Test was not guaranteed to make forward progress. Fixed with a 3 minute timeout.
2. Test was very slow in some conditions. Likewise.